frontend/awx/interfaces/JobTemplateForm.ts

```diff
@@ -1,61 +1,60 @@
-import { ExecutionEnvironment } from './ExecutionEnvironment';
 import { InstanceGroup } from './InstanceGroup';
-import { Inventory } from './Inventory';
-import { Label } from './Label';
-import { Project } from './Project';
-import { Credential } from './Credential';
+import { JobTemplate } from './JobTemplate';
 
-export interface JobTemplateForm {
+export interface JobTemplateSummaryFields
```

Comment:
I think you can just do `type JobTemplateSummaryFields = Omit<...>` rather than using `extends`. Same for `Pick` with JobTemplateRelatedFields  below